.github/workflows/ci.yaml

```diff
@@ -0,0 +1,64 @@
+name: Run tests
+
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - "*.md"
+      - "*.in"
+      - "*.txt"
+
+  pull_request:
+    branches: [main]
+    paths-ignore:
+      - "*.md"
+      - "*.in"
+      - "*.txt"
+
+jobs:
+  test:
+    env:
+      plugin_name: "otio_openassetio"
+
+    name: Test
+    strategy:
+      matrix:
+        python-version: [3.9, ]
+        os: [ubuntu-latest, macos-latest] # windows-latest when wheel is in place
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Clone OpenAssetIO
+        uses: actions/checkout@v2
+        with:
+          repository: TheFoundryVisionmongers/OpenAssetIO
+          path: dependencies/OpenAssetIO
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest pytest-cov -V OpenTimelineIO==0.13.0
+          pip install dependencies/OpenAssetIO
+
+      - name: Install Plugin
+        run: |
+          pip install -e .
+
+      - name: Lint with flake8
+        run: |
+          flake8 --show-source --statistics otio_openassetio tests
```

Comment:
[question]/[plan] Curious what you think of flake8 and if we should use it in openassetio?

Reply:
Haven't seen enough of it to form an opinion yet - was just sticking to the OTIO template for now to make the project more familiar to other OTIO plugin devs.